source/dash/graphics/adapters/sdl.d

```diff
@@ -103,7 +92,35 @@ public:
     override void resize()
     {
         loadProperties();
-        window.setSize( width, height );
+
+        bool bordered = false;
+        
+        switch( windowType )
+        {
+            case WindowType.Fullscreen:
+                trace("Setting main window to Fullscreen mode!");
+                window.setFullscreenSetting( SDL_WINDOW_FULLSCREEN );
+                break;
+            case WindowType.FullscreenWindowed:
+                trace("Setting main window to Fullscreen Windowed mode!");
+                bordered = false;
+                width = screenWidth;
+                height = screenHeight;
+                goto default;
+            case WindowType.Windowed:
+                trace("Setting main window to Windowed mode!");
+                bordered = true;
+                goto default;
+            case WindowType.BorderlessWindow:
+                trace("Setting main window to Borderless Windowed mode!");
+                bordered = false;
+                goto default;
+            default: 
+                window.setFullscreenSetting( 0 );
+                window.setSize( width, height );
+                SDL_SetWindowBordered( SDL_GetWindowFromID( window.id() ), bordered );
```

Comment:
This looks pretty grody. Is there any way to get a `window.setWindowBordered` (or similar) into `gfm`?